.travis.yml

```diff
@@ -81,12 +81,14 @@ jobs:
             <<: *STANDARD_TEST_JOB
             stage: Test
             php: 7.1
-            env: SYMFONY_DEPRECATIONS_HELPER=disabled PHP_CS_FIXER_TEST_USE_LEGACY_TOKENIZER=1 SYMFONY_VERSION="~4.1.0"
+            name: 7.1 | Symfony 4.1
+            env: SYMFONY_DEPRECATIONS_HELPER=disabled PHP_CS_FIXER_TEST_USE_LEGACY_TOKENIZER=1 SYMFONY_VERSION="^4.1"
```

Comment:
There's a subtle change here: now any 4.4.x version of Symfony can also be installed. Not sure this is desired.

Reply:
Reverted